src/main/webapp/app/entities/ttn/ttn-update.tsx

```diff
@@ -64,6 +66,8 @@ export class TTNUpdate extends React.Component<ITTNUpdateProps, ITTNUpdateState>
       transporterId: '0',
       isNew: !this.props.match.params || !this.props.match.params.id,
```

Comment:
Sometimes using the most spread library such as `lodash` allows as to simplify code. For example, `_.get(this, 'props.match.params.id')` - return `undefined` if `params === undefined` or `params.id === undefined` and we can eliminate one condition. (`_` is a lodash)